contrib/ci/test-swift.sh

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
+
+set -eof pipefail
+
+TEST_ROOT=$(dirname "${BASH_SOURCE}")/
+source "${TEST_ROOT}/test.sh"
+
+# make sure we are in this dir
+CURRENT_DIR=$(cd $(dirname $0); pwd)
+
+create-postgres-creds
+
+puts-step "fetching openstack credentials"
+
+# turn creds into something that we can use.
+mkdir -p $CURRENT_DIR/tmp/swift
+
+# guess which value to use for tenant:
+TENANT=""
+
+echo "test:tester" > $CURRENT_DIR/tmp/swift/username
+echo "testing" > $CURRENT_DIR/tmp/swift/password
+echo ${TENANT} > $CURRENT_DIR/tmp/swift/tenant
+echo "http://swift:8080/auth/v1.0" > $CURRENT_DIR/tmp/swift/authurl
+echo "1" > $CURRENT_DIR/tmp/swift/authversion
+echo "deis-swift-test" > $CURRENT_DIR/tmp/swift/database-container
+
+# kill containers when this script exits or errors out
+trap 'kill-container $SWIFT_DATA' INT TERM
+# boot swift
+SWIFT_DATA=$(docker run -v /srv --name SWIFT_DATA busybox)
+
+# kill containers when this script exits or errors out
+trap 'kill-container $SWIFT_JOB' INT TERM
+SWIFT_JOB=$(docker run --name onlyone --hostname onlyone -d -p 12345:8080 --volumes-from SWIFT_DATA -t kmala/swift-onlyone)
+
+
+# postgres container command
+PG_CMD="docker run -d --link $SWIFT_JOB:swift -e BACKUP_FREQUENCY=3s \
+         -e DATABASE_STORAGE=swift \
+         -v $CURRENT_DIR/tmp/creds:/var/run/secrets/deis/database/creds \
+         -v $CURRENT_DIR/tmp/swift:/var/run/secrets/deis/objectstore/creds \
+         $1"
+
+# kill containers when this script exits or errors out
+trap 'kill-container $PG_JOB' INT TERM
+start-postgres "$PG_CMD"
+
+
+
+# display logs for debugging purposes
+puts-step "displaying swift logs"
+docker logs $SWIFT_JOB
+
+check-postgres $PG_JOB
+
+# check if swift has some backups ... 3 ?
+puts-step "checking if swift has at least 3 backups"
+
+sudo apt-get install -y swift
```

Comment:
Any way we can just `docker exec` into the swift container to check the data instead of installing clients on the host?